updater/lib/dependabot/sorbet/runtime.rb

```diff
@@ -0,0 +1,29 @@
+require "sorbet-runtime"
```

Comment:
You might also need:

```suggestion
# typed: true
# frozen_string_literal: true

require "sentry-ruby"
require "sorbet-runtime"
```

Reply:
Since we already require Sentry before this file is required I don't think we need to require it here again?